array.c

```diff
@@ -2969,14 +2969,14 @@ inspect_ary(VALUE ary, VALUE dummy, int recur)
 
 /*
  *  call-seq:
- *    array.inspect -> new_string
+ *    inspect -> new_string
  *
- *  Returns the new String formed by calling method <tt>#inspect</tt>
- *  on each array element:
```

Comment:
This old documentation holds more information, especially the part where it calls `#inspect` on each element. We should keep that.

Reply:
Fixed.